README.md

```diff
@@ -477,7 +477,7 @@ Polymer 2.0 will continue to use a [shim](https://github.com/webcomponents/shady
 * <a name="breaking-dynamic-functions"> Declaring a method name used as an observer or computing function in the `properties` block causes the method property itself to become a dependency for any effects it is used in, meaning the effect for that method will run whenever the method is set, similar to 1.x. However, due to removing the `undefined` rule noted above, in 2.x if such a method exists when the element is created, it will run with initial values of arguments, even in the case some or all arguments are `undefined`.
 * <a name="breaking-binding-notifications">‘notify’ events are no longer fired when value changes _as result of binding from host_, as a major performance optimization over 1.x behavior.  Use cases such as `<my-el foo="{{bar}}" on-foo-changed="fooChanged">` are no longer supported.  In this case you should simply user a `bar` observer in the host.  Use cases such as dynamically adding a `property-changed` event listener on for properties bound by an element's host by an actor other than the host are no longer supported.
 * <a name="breaking-properties-deserialization"></a>In order for a property to be deserialized from its attribute, it must be declared in the `properties` metadata object
-* <a name="breaking-colleciton"></a>The `Polymer.Collection` and associated key-based path and splice notification for arrays has been eliminated.  See [explanation here](https://github.com/Polymer/polymer/pull/3970#issue-178203286) for more details.
+* <a name="breaking-collection"></a>The `Polymer.Collection` and associated key-based path and splice notification for arrays has been eliminated.  See [explanation here](https://github.com/Polymer/polymer/pull/3970#issue-178203286) for more details.
```

Comment:
This breaks link to the README. Do we know which pages link to this specific section?

Reply:
I hope by "we", you're asking the polymer team. And because it's an anchor, you won't be able to see it in your web server log file.

I certainly don't know :-( .

The best I can offer are the following:
* There are [no obvious hits](https://www.google.ca/search?q="breaking-colleciton"+polymer&oq="breaking-colleciton"+polymer).
* Unlike renaming a file, the worst outcome is that the user is at the top of the page, with the anchor in the urlbar. They could look for "collection" and would probably find their way to the right part of the document.

Reply:
Yeah, the "we" was targeting the Polymer team. It was just a general thought and I am not sure if we can answer this question. Let me ask some others :smile: